pkg/api/v1/summary_test.go

```diff
@@ -500,9 +508,11 @@ func TestListTabSummariesHTTP(t *testing.T) {
 						LatestPassingBuild:    "Lantern",
 						LastUpdateTimestamp: &timestamp.Timestamp{
 							Seconds: 916166800,
+							Nanos:   0,
```

Comment:
the default value for number fields in Go is 0, so you don't have specify in that case

Reply:
Got it!